bundles/org.openhab.binding.sunsynk/src/main/java/org/openhab/binding/sunsynk/internal/api/dto/Daytemps.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.sunsynk.internal.api.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * The {@link Daytemps} is the internal class for Inverter temperature history information
+ * from a Sun Synk Connect Account.
+ * The minute following midnight returns an empty array.
+ * 
+ * @author Lee Charlton - Initial contribution
+ */
+
+@NonNullByDefault
+public class Daytemps {
+    private int code;
+    private String msg = "";
+    private boolean success;
+    private Data data = new Data();
+    private boolean response_status = true;
```

Comment:
Many more in the DTO classes that should be adapted like this. They are also mentioned in the SAT error report.
```suggestion
    @SerializedName("response_status")
    private boolean responseStatus = true;
```

Reply:
Done